### PR TITLE
Clarify that constraints on reduced flags apply to non-layered images

### LINKED ISSUE #133: reduced_still_picture_header must be set to 0 for scalable images
Because reduced_still_picture_header forces the frame type to be a KEY_FRAME (and therefore all intra), it cannot be set for the dependent images in a scalable / progressive image. However, the current AVIF spec has in section 2.1:

> The AV1 Image Item Data should have its reduced_still_picture_header flag set to 1.

I suggest simply removing this line, allowing either 1 or 0 in AVIF.

### PR DESCRIPTION
**Pull request by cconcolato**
Closes #133

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/138.html" title="Last updated on Apr 1, 2021, 12:42 AM UTC (d458ee9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/138/68abca0...d458ee9.html" title="Last updated on Apr 1, 2021, 12:42 AM UTC (d458ee9)">Diff</a>

### CODE REVIEW

**Review by tdaede — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -169,8 +169,9 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
   - The content of an [=AV1 Image Item=] is called the <dfn>AV1 Image Item Data</dfn> and shall obey the following constraints:
     - The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].
     - The [=AV1 Image Item Data=] shall have exactly one [=Sequence Header OBU=].
-    - The [=AV1 Image Item Data=] should have its <code>[=still_picture=]</code> flag set to 1.
-    - The [=AV1 Image Item Data=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+    - If the [=AV1 Image Item Data=] is made of a single frame (i.e. when using a single layer),
```
Comment (wantehchang) [inline index.bs:172]: Nit: "made up of" sounds better than "made of" here.

File: index.bs [hunk 2]
```diff
@@ -169,8 +169,9 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
...
+      - It should have its <code>[=still_picture=]</code> flag set to 1.
+      - It should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
```
Comment (leo-barnes) [inline index.bs:174]: Feels like these two lines should be indented one more step

File: index.bs (hunk 1 above)
Comment (leo-barnes) [reply index.bs:172]: Or "consists of"?

**Review by leo-barnes — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (wantehchang) [reply index.bs:172]: Yes, I like "consists of" better.

**Comment by leo-barnes** [comment]
Looks good!

**Pull request merged**